andela/ah-the-unsullied-frontend
# #164202260 Redirect http:// links  to https://

#### What does this PR do?
-Redirects http links to https 
#### Description of Task to be completed?
-Automatically redirects http links to https unless page is loaded from a web server running on localhost
#### How should this be manually tested?
1. Clone the repository
```
git clone https://github.com/andela/ah-the-unsullied-frontend.git
```
2. Cd into the cloned repo
```
cd ah-the-unsullied-frontend
```
3. Checkout to this branch
```
git checkout ft-tags-162949181 
```
4. Install the dependencies of the project
```
npm install
```
5. Run the app
```
npm start
```
6. Visit the default route `http://localhost:3000/` 

7. To run tests, run `npm test`

7. To test the links deploy the project in heroku and run the correct link without the `https://` or with `http` in the url it should redirect you to the link with `https://`

#### Any background context you want to provide?
Cannort redirect localhost links
#### What are the relevant pivotal tracker stories?
 #164202260 https://www.pivotaltracker.com/story/show/164202260
#### Screenshots (if appropriate)
Enter the link without https://
![image](https://user-images.githubusercontent.com/28457081/53580915-abc8c580-3b8d-11e9-9988-7fc2da70669e.png)

It should redirect you to one with https://
![image](https://user-images.githubusercontent.com/28457081/53580943-b5522d80-3b8d-11e9-83d6-0801915046de.png)

#### Questions:
